src/views/ifsBrowser.js

```diff
@@ -636,12 +636,19 @@ class Object extends vscode.TreeItem {
    * @param {"shortcut"|"directory"|"streamfile"} type
    * @param {string} label
    * @param {string} path
+   * @param {number} size
+   * @param {date} modified
```

Comment:
```suggestion
   * @param {date} [modified]
```

Same here for optional. But what is `date`? Is it really a `number` or an action JS `Date` type?